cli/command_restore.go

```diff
@@ -343,6 +375,12 @@ func (c *commandRestore) run(ctx context.Context, rep repo.Repository) error {
 			rootEntry = re
 		}
 
+		// TODO(rjk): error handling... plumbing, etc.
+		// I don't think that
+		if err := c.getDotIgnoreFilesFromPolicy(ctx, rep, rstp); err != nil {
```

Comment:
So this should work but only partially, because one can have dotignore files as child policies. For example:

foo@bar:/home policy to ignore `.kopiaignore`
foo@bar:/home/jarek policy to ignore `.jarekignore` (which may or may not inherit `.kopiaignore`)

(that means that `/home/rjk` will have `.kopiaignore` while `/home/jarek` will have `.kopiaignore` and `.jarekignore`

IOW i think you need to plumb through `*policy.Tree` instead of statically reading the policy at the root just like during upload.